tensorboard/webapp/widgets/data_table/data_table_component.ts

```diff
@@ -281,4 +295,11 @@ export class DataTableComponent implements OnDestroy {
       'highlight-border-left': this.highlightSide === Side.LEFT,
     };
   }
+
+  showColumn(header: ColumnHeader) {
+    return (
+      header.enabled &&
+      (this.smoothingEnabled || header.type !== ColumnHeaderType.SMOOTHED)
```

Comment:
Could we just "disable" the SMOOTHED column when smoothing is disabled and avoid this check? It would be fine to make this change in another PR btw.

Reply:
The Smoothed column being disabled is different from it not being shown. When disabled it will not appear when the smoothing factor is set.  However, as we discussed offline we plan to leave the SMOOTHED column in even when smoothing is not enabled. This change will come in a future PR.